src/index.html

```diff
@@ -1,14 +1,79 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <header class="header">
+
+      <img
+      src="images/logo.png"
+      alt="Лого интернет-магазина Mojo"
+      class="image"
+      >
```

Comment:
```suggestion
      <img
        src="images/logo.png"
        alt="Лого интернет-магазина Mojo"
        class="image"
      >
```
Would be better to wrap this 'img' inside 'a'-tag. Usually. clicking on the logo leads to the homepage